auth/src/main/res/layout/fui_auth_method_picker_layout.xml

```diff
@@ -26,12 +27,23 @@
         app:layout_constraintEnd_toEndOf="parent"
         app:layout_constraintTop_toTopOf="parent"
         app:layout_constraintBottom_toBottomOf="parent"
-        app:layout_constraintVertical_bias="0.9">
+        app:layout_constraintVertical_bias="0.7">
 
         <LinearLayout
             android:id="@+id/btn_holder"
-            style="@style/FirebaseUI.AuthMethodPicker.ButtonHolder" />
-
+            style="@style/FirebaseUI.AuthMethodPicker.ButtonHolder"/>
     </ScrollView>
-
+    <TextView
+        android:id="@+id/main_tos_and_pp"
+        style="@style/FirebaseUI.Text.BodyText"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginTop="@dimen/fui_field_padding_vert"
+        android:textColor="?android:textColorTertiary"
```

Comment:
Does the color change if you remove this? Either way, we need to use some AppCompat version, not the system one. I'll get back to you if the color is wrong when you remove it.

Reply:
My comment above addresses this I think.

Reply:
Changed it to attr/textColorTertiary